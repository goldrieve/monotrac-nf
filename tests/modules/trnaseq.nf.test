nextflow_process {

    name "Test Process TRANSEQ"
    script "main.nf"
    process "TRANSEQ"

    test("Should run transeq") {

        when {
            params {
            }
            process {
                """
                input[0] = ["SVP_test", file("${projectDir}/tests/data/SVP.fas")] 
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
