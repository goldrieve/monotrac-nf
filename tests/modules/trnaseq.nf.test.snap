{
    "Should run transeq": {
        "content": [
            {
                "0": [
                    [
                        "SVP_test",
                        "SVP_test_AA.fas:md5,b452ecefb384568e0e4be0769adf4722"
                    ]
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.7"
        },
        "timestamp": "2025-10-06T14:40:02.396028079"
    }
}